### DISCUSSION TITLE
Allow game authors to direct voters feedback efforts

### DISCUSSION THREAD

**Discussion by sgstair** [ludumdare/ludumdare#2032] [Ideas]
A few suggestions have come up on the LD39 results thread asking for effectively some way to get something more specific out of their voters. Such as a "Looking for feedback on: (user defined text)" (suggested by seconddimension), or as a game-author-defined poll to get feedback on something the author feels is important (Initially suggested by zelos)

I think this kind of thing is a low hanging fruit that could be beneficial to LD and let game authors get more out of the experience, so we should investigate adding some kind of mechanism along these lines.

**Comment by mikekasprzak** [comment]
Interesting. So effectively changing the voting stage to a sort of Q&A. Your ratings, plus any additional questions the author has for the player.

**Comment by local-minimum** [comment]
This would really be intriguing. A user may add their own voting category for only their game, or a question that the raters can choose to vote and comment on. I see some issues with how the database is structured but as a feature this would make much sense and be fun. It would be important that the question can't be changed after first answer on it.

**Comment by sgstair** [comment]
We could add simple voting-like Q&A I think by adding some "custom" grades that don't get aggregated across all games, and using game node meta to store what the questions should be, possibly with UI similar to the existing link UI. That would be a pretty compelling start I think

**Comment by mikekasprzak** [comment]
Since the custom results aren't going to be directly comparable across users, it doesn't necessarily need to be in the grade table. Might as well have a mega response type (number and/or string) for polls.

**Comment by local-minimum** [comment]
We could easily implement custom grading (could suggest from previous customs of other ppl), and collect that in a table similar to the grade table. Then we could also implement custom Q&A, with different table. They don't really depend on each other. This would also be good if we ever decide to drop one of the current categories, then those who want to keep a voting category can just custom it.

**Comment by mikekasprzak** [comment]
Better question: what are some things people would actually ask? I'm not
seeing much need for "custom grades" when you can't actually compare them
to anything.

**Comment by local-minimum** [comment]
Sorry, perhaps I need to elaborate. If I wanted a category like `difficulty` or `intuitiveness` or other, these could at the end of the event be tallied down to a score just like the standard voting categories. Reason to have them as voting category and not as comment would be that more people tend to give grades than comment and that you would get an average result in the end. If we also let these categories be globally known (kind of like a tag) then many users could use the same category. For example maybe I decided I want a snowman-unicode category for this winter LD and I promote it so that a lot of people pick it up. Then it could be treated like all other categories. So that at the end of the event, you would get a rank in the bonus category you joined. Even if there were just 3 other games in the same category, it could still be fun to look at those and see why they had the same category and how they did. The site don't need to award any trophies nor promote any of the winners. But if people found bonus categories  to be a fun thing, the winners of the most commonly used bonus-categories could also be promoted in their own section. Just like ordinary winners.

So tldr: It could encourage playfulness and could be a nice way to get focused statistics about something you care extra about.

**Comment by mikekasprzak** [comment]
Okay, so you _do_ want to be able to compare them.

I kinda feel like this is a separate item. I like the term "Bonus Categories".

I don't want to dwell too much on this at the moment, but I know there are a bunch of issues that would have to be addressed here (minimum votes vs regluar judging, performance to calculate said scores, etc). Anyway, this should be its own issue.

**Comment by local-minimum** [comment]
Agree!
